source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -276,6 +279,18 @@ def _get_mode_weight(journey):
     return max(mode_weight.get(mode, 1) for mode in journey.tags)
 
 
+def get_min_waiting(journey):
+    """
+    Returns min waiting time in a journey
+    """
+    waiting = 3600*4
```

Comment:
but this value is not relevant for your method (and if we change the max waiting value we'll forget to update this method :wink: and a journey with no sections will have a 4h waiting)


and I think it will be more pythonic to write this (not tested): 
```python
return 0 if not journey.sections 
             else min(s.duration for s in journey.sections if s.type == response_pb2.WAITING)
```